### DISCUSSION TITLE
快照版的更新又不弹窗了（笑

### DISCUSSION THREAD

**Discussion by kong-que-dnf123** [Meloong-Git/PCL#4653] [提问] [answered]
一更来一趟 趟趟都一样 更新弹窗又汐了（（（
~这算了我不发ISSUE了（（（~
（我承诺此次PCL是我刚刚打开电脑并启动PCL 不存在长期休眠导致自更新检查受时间波动影响 以及此次日志为我第二次尝试）
（注：图片之后为日志 且只有log1 log2被重写过）
![image](https://github.com/user-attachments/assets/13613e61-d373-43bd-b419-c42b335a6954)
![image](https://github.com/user-attachments/assets/e39a1b02-ae64-469b-9de9-dbd71c505a5a)
![image](https://github.com/user-attachments/assets/dd406307-2a55-4a02-a2b0-e211781c108f)
![image](https://github.com/user-attachments/assets/58cdfc83-13a4-46c9-bf22-b6a4df6f7e58)
[Log1.txt](https://github.com/user-attachments/files/16822791/Log1.txt)
[Log2.txt](https://github.com/user-attachments/files/16822792/Log2.txt)
[Log3.txt](https://github.com/user-attachments/files/16822793/Log3.txt)
[Log4.txt](https://github.com/user-attachments/files/16822794/Log4.txt)
[Log5.txt](https://github.com/user-attachments/files/16822795/Log5.txt)

**Comment by MoYuan-CN** [comment, marked as answer]
我感觉是检查更新的 API 被缓存了导致的

**Comment by kong-que-dnf123** [comment]
被缓存补充：
2.8.6更新给了2.8.5的公告.jpg（
![image](https://github.com/user-attachments/assets/c0518d52-577f-4400-802f-2d5a7ffa8422)
[Log2.txt](https://github.com/user-attachments/files/16889701/Log2.txt)

**Reply by tangge233** [reply]
因为正式版和预览版一同更新（
~~公告用的同一个~~

**Comment by iPodtouch6557** [comment]
我这边发现确实会出现这种问题，因为我导出错误报告的时候报错了才发现没更新，并且是隔了半个月全新的启动，不存在休眠问题

**Reply by 3gf8jv4dv** [reply]
https://github.com/Hex-Dragon/PCL2/issues/3862#issuecomment-2330177591
